style/common/common.css

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
Remove this file from the commit. (You can do this by doing a "git rm style/common/common.css" and committing.)